docker-jans-casa/Dockerfile

```
@@ -1,3 +1,27 @@
+# =================================
+# Cloud SQL JDBC Socket Factory Builder
+# =================================
+# Builds fat JARs with all dependencies for Cloud SQL connectivity
+FROM maven:3.9-eclipse-temurin-17 AS cloudsql-builder
+
+ARG CLOUDSQL_SOCKET_FACTORY_VERSION=1.27.0
+
+RUN apt-get update && apt-get install -y --no-install-recommends git \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)